packages/server/postgres/migrations/2025-09-02T15:57:49.742Z_addSharePageNotificationsCheck.ts

```diff
@@ -0,0 +1,16 @@
+import {type Kysely, sql} from 'kysely'
+
+// `any` is required here since migrations should be frozen in time. alternatively, keep a "snapshot" db interface.
+export async function up(db: Kysely<any>): Promise<void> {
+  // cannot be in the same migration as adding the enum type, as it's unsafe to use the enum in the same transaction as it's created
```

Comment:
It event needs to be in a different `migrate:latest` run, because all migrations run at once share the same transaction, so the enum is still considered unsafe.